lib/libbol.py

```diff
@@ -702,6 +703,16 @@ def write(self, f):
             f.write(pack(">h", self.userdata[i]))
         #assert f.tell() - start == self._size
 
+    def route_info(self):
+        if not self.objectid in OBJECTNAMES:
+            return
+        name = OBJECTNAMES[self.objectid]
+        filename = join("object_parameters", name+".json")
+        if not exists(filename):
```

Comment:
Technically, it should be [`os.path.isfile()`](https://docs.python.org/3/library/os.path.html#os.path.isfile). [`os.path.exists()`](https://docs.python.org/3/library/os.path.html#os.path.exists) is more general; we are looking for a file here.